src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -147,4 +147,11 @@ private static JWTVerifier getVerifier(Request req, String token) {
     public static boolean authDisabled() {
         return hasConfigProperty("DISABLE_AUTH") && "true".equals(getConfigPropertyAsText("DISABLE_AUTH"));
     }
+
+    /**
+     * Computes a token expiration time in seconds.
+     */
+    public static long getTokenExpirationTime(TokenHolder token) {
+        return System.currentTimeMillis() + token.getExpiresIn() /* seconds */ * 1000;
+    }
```

Comment:
Looks like I missed this. Could you delete, please?

Reply:
Thanks for catching that. I just removed it.